Source/core/CarbonFootprintCore.js

```diff
@@ -58,13 +58,14 @@ CarbonFootprintCore.prototype.createFootprintElement = function (distance) {
     var e = document.createElement('div');
 
     var treesStr = this.treesToString(this.computeTrees(footprint));
+    var knowMoreUrl = chrome.extension.getURL("pages/knowMore.html");
 
-    e.innerHTML = '<a href=\'http://goo.gl/yxdIs\' target=\'_blank\' title=\'' +
+    e.innerHTML = '<a href=' + knowMoreUrl + ' target=\'_blank\' title=\'' +
         treesStr +
         '\' class=\'carbon\' id=\'carbon\'>' +
         this.footprintToString(footprint) +
-        '</a> <a class=\'offset-link\' href=\'http://goo.gl/yxdIs\' target=\'_blank\' title=\'' +
-        treesStr + '\'>offset</a>';
+        '</a> <a class=\'know-more-link\' href=' + knowMoreUrl + ' target=\'_blank\' title=\'' +
+        treesStr + '\'>Know More</a>';
```

Comment:
Perhaps we could use a question mark icon instead of the text "Know More". What do you think?